test/e2e/e2e-service-provider/src/main/resources/application.yml

```diff
@@ -32,3 +32,17 @@ spring:
     properties:
       hibernate.format_sql: true
     show-sql: true
+
+management:
+  metrics:
+    use-global-registry: true
+    web:
+      server:
+        request:
+          autotime:
+            enabled: true
+  endpoints:
+    web:
+      exposure:
+        include: info,health
+      base-path: /
```

Comment:
Why expose these endpoints? The `info` endpoint conflict with the example service, if they're really needed, please expose them to another `base-path`

Reply:
Thanks a lot. Has fixed.